cmd/monitor/monitor.go

```diff
@@ -167,58 +172,8 @@ func (h historicalRange) getValues(limit int) []float64 {
 	}
 	return values
 }
-func prependLatestBlocks(ctx context.Context, ms *monitorStatus, rpc *ethrpc.Client) {
-	from := new(big.Int).Sub(ms.HeadBlock, big.NewInt(int64(batchSize-1)))
-	// Prevent getBlockRange from fetching duplicate blocks.
-	if ms.MaxBlockRetrieved.Cmp(from) == 1 {
-		from.Add(ms.MaxBlockRetrieved, big.NewInt(1))
-	}
 
-	if from.Cmp(zero) < 0 {
-		from.SetInt64(0)
-	}
-
-	log.Debug().
-		Int64("from", from.Int64()).
-		Int64("to", ms.HeadBlock.Int64()).
-		Int64("max", ms.MaxBlockRetrieved.Int64()).
-		Msg("Fetching latest blocks")
-
-	err := ms.getBlockRange(ctx, from, ms.HeadBlock, rpc)
-	if err != nil {
-		log.Error().Err(err).Msg("There was an issue fetching the block range")
-	}
-}
-
-func appendOlderBlocks(ctx context.Context, ms *monitorStatus, rpc *ethrpc.Client) error {
-	if ms.MinBlockRetrieved == nil {
-		log.Warn().Msg("Nil min block")
-		return fmt.Errorf("the min block is nil")
-	}
-	if !currentlyFetchingHistoryLock.TryLock() {
-		return fmt.Errorf("the function is currently locked")
-	}
-	defer currentlyFetchingHistoryLock.Unlock()
-
-	to := new(big.Int).Sub(ms.MinBlockRetrieved, one)
-	from := new(big.Int).Sub(to, big.NewInt(int64(batchSize-1)))
-	if from.Cmp(zero) < 0 {
-		from.SetInt64(0)
-	}
-
-	log.Debug().
-		Int64("from", from.Int64()).
-		Int64("to", to.Int64()).
-		Int64("min", ms.MinBlockRetrieved.Int64()).
-		Msg("Fetching older blocks")
-
-	err := ms.getBlockRange(ctx, from, to, rpc)
-	if err != nil {
-		log.Error().Err(err).Msg("There was an issue fetching the block range")
-		return err
-	}
-	return nil
-}
+const maxDataPoints = 1000
```

Comment:
I think we should put this value at the top of the file with a comment that explains exactly what it does. I can imagine myself checking the code in 3 months wondering what this is about :)